wrangler-transform/src/main/java/co/cask/wrangler/Wrangler.java

```diff
@@ -542,35 +542,33 @@ private URL getDPServiceURL(String method) throws URISyntaxException, MalformedU
     private String directives;
 
     @Name("udd")
-    @Description("List of User Defined Directives (UDD) that have to be loaded.")
+    @Description("List of User Defined Directives (UDD) that have to be loaded")
     @Nullable
     private String udds;
 
     @Name("field")
-    @Description("Name of the input field to be wrangled or '*' to wrangle all the fields.")
+    @Description("Name of the input field to be wrangled or '*' to wrangle all the fields")
     @Macro
     private final String field;
 
-    @Name("threshold")
-    @Description("Max number of event failures in wrangling after which to stop the pipeline of processing. " +
-      "Threshold is not aggregate across all instance, but is applied for each running instances. " +
-      "Set to -1 to specify unlimited number of acceptable errors.")
-    @Macro
-    private final int threshold;
-
     @Name("schema")
-    @Description("Specifies the schema that has to be output.")
+    @Description("Specifies the schema that has to be output")
     @Macro
     private final String schema;
 
+    @Name("on-error")
+    @Description("How to handle error in record processing")
+    @Macro
+    private final String onError;
```

Comment:
Can we make this property nullable? It will break UI for a lot of the default configurations

Reply:
Not sure why it would break ? Could you elaborate.

Reply:
Also, we don't want it to nullable. We have default, so not sure were it would cause problem and why.